bitnami/nginx/Chart.yaml

```diff
@@ -34,4 +34,4 @@ maintainers:
 name: nginx
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/nginx
-version: 15.5.3
+version: 15.6.0
```

Comment:
Please merge latest changes and fix the conflicts. This version is not valid anymore (sorry for our late response)